Fixes issue 6234 by updating documentation for Cubical Agda and removing links (#6977)
agda/agda: doc/user-manual/language/cubical.lagda.rst

6. Higher inductive types
7. Cubical identity types

There are two major libraries for Cubical Agda:

- ``agda/cubical``: originally intended as a standard library for
  Cubical Agda available at https://github.com/agda/cubical. This
  documentation uses the naming conventions of this library, for a
  detailed list of all of the built-in Cubical Agda files and
  primitives see :ref:`primitives-ref`.

- ``1lab``: A formalised and cross linked reference resource for
  cubical methods in Homotopy Type Theory which can be found at
  https://1lab.dev/. Much better documented than the ``agda/cubical``

>   library and hence more accessible to newcomers. The sources can be

Reviewer: :blue_heart:

  open import Cubical.Core.Everything

> For detailed install instructions for ``agda/cubical`` see:

Author: ```suggestion
Follow the instructions at https://github.com/agda/cubical to install the library.
```

https://github.com/agda/cubical/blob/4de6b6939245ce281f02e47af02f2deb1cbd853e/INSTALL.md. In

> order to make this library visible to Agda add

Reviewer 2: I don't think this is better - if we link to specific commit, we should keep this link updated. Shouldn't we just say that people should follow the instructions at https://github.com/agda/cubical to install cubical?

Author: ```suggestion
In order to make this library visible to Agda add
```

Author: Thanks, good point! Fixed it the way you suggested and deleted the link
